go/vt/sqlparser/parse_test.go

```diff
@@ -3037,6 +3037,21 @@ func TestKeywords(t *testing.T) {
 		}, {
 			input:  "select variables from t",
 			output: "select `variables` from t",
+		}, {
+			input:  "select 1 as found",
+			output: "select 1 as `found` from dual",
+		}, {
+			input:  "select found from t",
+			output: "select `found` from t",
+		}, {
```

Comment:
it would be good to automate something where we generate the variety of queries that use idents -- table and subquery aliasing, windows, ctes, DDL, DML, etc

Reply:
that might go better in sqllogictests